include/electionguard/election.hpp

```diff
@@ -113,6 +113,8 @@ namespace electionguard
         /// </summary>
         const ElementModP *getElGamalPublicKey() const;
 
+        const ElementModP &getElGamalPublicKeyRef() const;
```

Comment:
TODO: cleanup

Reply:
was having some issues accessing the pointer version from typescript so added a reference version.